src/config/mod.rs

```diff
@@ -462,14 +488,55 @@ pub struct TestInput {
     pub metric: Option<Metric>,
 }
 
+#[derive(Deserialize, Serialize, Debug, Clone, PartialEq, Eq, Hash)]
+#[serde(untagged)]
+pub enum OneOrMany<T> {
```

Comment:
we could probably move this to another place to use it in another place, but where?

Reply:
The advantage is that we don't have a breaking change regarding the unit tests having an `etract_from` becoming a `Vec`.